### ISSUE TITLE
No such file or directory

### ISSUE THREAD

**Issue by chengyalong123** [ThomasWestfechtel/GSDE#2]
Sir, first of all thank you very much for answering my first question, but there is one more question. 
snapshot/O31_1_1_1/42/A2W/logits-3.npy' This file seems to be loaded directly without being generated in the code.

![2](https://github.com/ThomasWestfechtel/GSDE/assets/125198230/28f88468-afaf-4469-834b-1502ecac9555)
![1](https://github.com/ThomasWestfechtel/GSDE/assets/125198230/bfee8699-19e9-4702-b25b-82dbbd9b280e)
![3](https://github.com/ThomasWestfechtel/GSDE/assets/125198230/6b87e433-81b1-465b-9117-6a81cc14a002)

**Comment by ThomasWestfechtel** [comment]
The logit file contains the results from the previous run.
In order to run for run_id of 4 you need to run the algorithm first for run_id of 0, then 1, then 2, and then 3.
The logit file is saved at line 565 in the main.py file 

![labels](https://github.com/ThomasWestfechtel/GSDE/assets/37787633/d02d9f83-8825-4dc4-8343-50e3734972bb)
